applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -295,7 +295,13 @@ static int initialize(void)
 	create_param.params_count = ARRAY_SIZE(subgroup_params);
 	create_param.params = subgroup_params;
 	create_param.qos = &lc3_preset.qos;
-	create_param.packing = BT_ISO_PACKING_SEQUENTIAL;
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_INTERLEAVED)) {
```

Comment:
I think we should at least stick to the same scheme in all the application files.
Doing #if here would save code space, and CONFIG_BT_AUDIO_INTERLEAVED will never change run-time. Or am I missing something?
My impression is that we have been doing #if for CONFIGs unless this causes compilation errors that cant be avoided.

Reply:
Searching for 'if (IS_ENABLED(' gives me 16 hits in our folders, whereas '"if CONFIG' gives me 27, so there is a mix already.

Reply:
I think that if we don't test both options as part of the CI system, the 'if (IS_ENABLED(' is better since the code is there and will be compiled. Meaning that if BT_ISO_PACKING_INTERLEAVED suddenly disappeared from zephyr, our CI system would pick that up. However, if we use #if, then it can disappear and no one would know until it is enabled again.

Reply:
It is probably better to have a discussion on Teams.
"#if CONFIG" gives 27 results and "#if (CONFIG" gives 60 (this is another inconsistency we have). So a total of 87.